documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,216 @@
+* Ensuring that there are enough xref:con-common-configuration-resources-reference[CPU and memory resources]
+* Increasing the number of tasks that run in parallel by doing the following:
+** Increasing the number of tasks that run in parallel using the `tasksMax` property
+** Increasing the number of worker nodes that run tasks using the `replicas` property
+
+Consider the number of tasks that can run in parallel according to the available CPU and memory resources and number of worker nodes. 
+You might need to keep adjusting the configuration values until they have the desired effect.
+
+== Configuring Kafka Connect for high-volume messages
+Kafka Connect fetches data from the source external data system and hands it to the Kafka Connect runtime producers so that it's replicated to the target cluster.
+
+The following example shows configuration for Kafka Connect using the `KafkaConnect` custom resource. 
+
+.Example Kafka Connect configuration for handling high volumes of messages
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaConnectApiVersion}
+kind: KafkaConnect
+metadata:
+  name: my-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+  replicas: 3
+  config:
+    offset.flush.timeout.ms: 10000
+    # ...
+  resources:
+    requests:
+      cpu: "1"
+      memory: 2Gi
+    limits:
+      cpu: "2"
+      memory: 2Gi
+  # ...
+----
+
+Consumer and producer configuration is added for the source connector, which is managed using the `KafkaConnector` custom resource. 
+
+.Example KafkaConnector configuration for handling high volumes of messages
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaConnectorApiVersion}
+kind: KafkaConnector
+metadata:
+  name: my-source-connector
+  labels:
+    strimzi.io/cluster: my-connect-cluster
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector
+  tasksMax: 2
+  config:
+    producer.override.batch.size: 327680
+    producer.override.linger.ms: 100
+    # ...  
+----
+
+If you are using the Kafka Connect API instead of the `KafkaConnector` custom resource to manage your connectors, you can add the connector configuration as a JSON object. 
+
+.Example curl request to add connector configuration for handling high volumes of messages
+[source,curl,subs=attributes+]
+----
+curl -X POST \
+  http://my-connect-cluster-connect-api:8083/connectors \
+  -H 'Content-Type: application/json' \
+  -d '{ "name": "my-source-connector",
+    "config":
+    {
+      "connector.class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
+      "file": "/opt/kafka/LICENSE",
+      "topic":"my-topic",
+      "tasksMax": "4",
+      "type": "source"
+      "producer.override.batch.size": 327680
+      "producer.override.linger.ms": 100
+      "consumer.fetch.max.bytes": 52428800
+      "consumer.max.partition.fetch.bytes": 1048576
+      "consumer.max.poll.records": 500
```

Comment:
The producers and consumers in a single example seem a bit wierd. Maybe if we want to show example of both options, we should make it universal for both sink and source connectors (replace the connector class and type with some placeholders for example).

Reply:
I thought it might be useful to show the consumer config in a sink connector example.

Reply:
It would. But I think we should have two examples for better understanding or make the example more generic by having some placeholders / comments instead of the connector type and name etc.